core/src/com/unciv/logic/map/MapGenerator.kt

```diff
@@ -659,6 +667,9 @@ class MapGenerator(val ruleset: Ruleset) {
             return Perlin.noise3d(worldCoords.x.toDouble(), worldCoords.y.toDouble(), seed, nOctaves, persistence, lacunarity, scale)
         }
 
+        /**
+         * Generates ridged perlin noise. As for parameters see [getPerlinNoise]
```

Comment:
A link explaining the basic principle behind ridged perlin noise https://stackoverflow.com/questions/36796829/procedural-terrain-with-ridged-fractal-noise